server.js

```diff
@@ -13,36 +14,59 @@ app = require('./lib/socketServer.js')(app, { verbose: false })(
 
 let osc = require('./lib/oscServer.js')(room.client('osc'))
 
-app.listen(PORT)
+const services = new Map([
```

Comment:
You should document what this is supposed to hold... kind of the type signature of the thing. Also it might make more sense for there to be a `services.draw()` method, and etc here and you can factor this out into its own file and class?